spec/telegraf/rack_spec.rb

```diff
@@ -101,4 +105,27 @@
       end
     end
   end
+
+  context 'with before_send filter' do
+    let(:args) { {before_send: before_send} }
+
+    context 'dropping rack paths' do
+      let(:before_send) do
+        lambda {|point, request:, **|
+          return if %r{^/healthcheck}.match?(request.path)
+
+          point.values[:path] = request.path
+          point
+        }
+      end
+
+      it('drops matching points') do
```

Comment:
Nitpick... Hey, you're mixing up the DSL styles. 🤓 

```suggestion
      it 'drops matching points' do
```

Here and elsewhere. 🙈